salt/renderers/gpg.py

```diff
@@ -431,6 +453,12 @@ def render(gpg_data, saltenv="base", sls="", argline="", **kwargs):
         raise SaltRenderError("GPG unavailable")
     log.debug("Reading GPG keys from: %s", _get_key_dir())
 
+    if not __opts__["gpg_decrypt_must_succeed"]:
```

Comment:
I'm worried that adding a deprecation warning whenever the files are rendered would be too noisy. Can we add the deprecation notice when its not able to decrypt?